gammapy/datasets/map.py

```diff
@@ -1841,29 +1842,35 @@ def cutout(self, position, width, mode="trim", name=None):
             Cutout map dataset.
         """
         name = make_name(name)
-        kwargs = {"gti": self.gti, "name": name, "meta_table": self.meta_table}
-        cutout_kwargs = {"position": position, "width": width, "mode": mode}
+        kwargs = {
+            "gti": self.gti,
+            "name": name,
+            "meta_table": self.meta_table,
+            "meta": self.meta,
```

Comment:
I think we should deprecate meta_table, but we have not done that yet in the datasets metadata

Reply:
If I remember correctly we kept the `meta_table` for now, because we have not settled on a meta data structure for for multiple datasets yet (where the table comes in handy...). See https://github.com/gammapy/gammapy/pull/4853#issuecomment-1906605913 for some more thoughts...